lib/pure/collections/intsets.nim

```diff
@@ -7,13 +7,23 @@
 #    distribution, for details about the copyright.
 #
 
-## The ``intsets`` module implements an efficient `int` set implemented as a
+## The ``intsets`` module implements an efficient ordinal set implemented as a
```

Comment:
what about instead adding `lib/pure/collections/ordsets.nim` (with no mention of intsets there) and then making  `lib/pure/collections/intsets.nim` a thin wrapper over `OrdSet[int]` that imports ordsets ?

Reply:
I thought about that, but I wanted to keep it as simple as possible to spot the differences, does github detect moved to show the diff?

Reply:
Now that I think of it, with the move in a separate commit the diff is still seen in the first commit, so I did the move to ordsets and kept intsets as an alias to ordsets. Not sure if that causes any issue with backwards compatibility. This is my first nim code,

Reply:
Still, there are lots of imports in the compiler. is there any way of importing transitively in nim? i.e. I leave intset just with the IntSet type alias and IntSet specific procs and import from ord. It seems that it some old code imports intset it won't get the other procs transitively even if intset imports from ordset, is this correct?

Reply:
did you try this?
```nim
# intsets.nim
import std/ordsets
export ordsets
```

Reply:
Ok, I didn't know about exports. However it says it 'Error: cannot export: ordsets'

Reply:
why? do you have a minimal example (or simply a temporary branch in your nim fork ) where it gives this error so we can look into it?
for eg this works and is a common pattern:
```nim
# in foo.nim
import std/tables
export tables
```